app/src/main/java/org/schabi/newpipe/player/event/PlayerGestureListener.java

```diff
@@ -69,26 +69,20 @@ public void onSingleTap(@NotNull final MainPlayer.PlayerType playerType) {
         if (DEBUG) {
             Log.d(TAG, "onSingleTap called with playerType = [" + player.getPlayerType() + "]");
         }
-        if (playerType == MainPlayer.PlayerType.POPUP) {
 
-            if (player.isControlsVisible()) {
-                player.hideControls(100, 100);
-            } else {
-                player.getPlayPauseButton().requestFocus();
-                player.showControlsThenHide();
-            }
+        if (player.isControlsVisible()) {
 
-        } else /* playerType == MainPlayer.PlayerType.VIDEO */ {
 
```

Comment:
There are still two misleading newlines separating the `if` from its body

Reply:
Done